subiquity/common/filesystem/gaps.py

```diff
@@ -260,6 +277,19 @@ def movable_trailing_partitions_and_gap_size(partition):
     return (trailing_partitions, 0)
 
 
+@movable_trailing_partitions_and_gap_size.register
+def _movable_trailing_partitions_and_gap_size_lvm(volume: LVM_LogicalVolume) \
+        -> Tuple[List[LVM_LogicalVolume], int]:
+    # In a Volume Group, there is no need to move partitions around, one can
+    # always use the remaining space.
+
+    def is_gap(x):
+        return isinstance(x, Gap)
+
+    gap = next(filter(is_gap, parts_and_gaps(volume.volgroup)), None)
+    return ([], gap.size if gap is not None else 0)
```

Comment:
isn't this size calculation the same as "largest_gap_size(volume.volgroup)"?

Reply:
Good point. I guess theoretically it should be, yes. I'll update this function.

AFAIK, from LVM standpoint, there should never be multiple gaps in a volume group, so the largest gap (if any if found) should always be the right one.

I am not familiar with our utilization of Gap objects though ; so I'd assume that:
 * Gap objects of size 0 are not a thing. If they are though, using `largest_gap_size` will do a better job.
 * Two successive Gap objects (which could/should be merged into one) are not a thing.

Reply:
> Gap objects of size 0 are not a thing. 

correct

> Two successive Gap objects (which could/should be merged into one) are not a thing.

99% correct.  We can split a gap moments before handing a gap off for creation of a partition, and in that moment there are 2 successive gaps, but the part_and_gaps will never respond in that way.

Reply:
Thanks! Updated.